<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Select">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>83</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>720</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="View">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>410</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Run_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="menscare">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>맨즈케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="skincare">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>스킨케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="makeup">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>메이크업</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bodycare">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>바디케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="haircare">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>헤어케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="beauty_tool">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>뷰티툴</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="maskpack">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>마스크/팩</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cleansing">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>클렌징</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="suncare">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fragrance">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>프래그런스</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All Products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Upload">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Time">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Products">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>상품 개체 &amp; 리뷰 수집</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>신규 상품 수 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>신규 상품코드 수집</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>예상 소요 시간 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
